internal/graph/cached_resolver.go

```diff
@@ -149,16 +158,23 @@ func (c *CachedCheckResolver) ResolveCheck(
 		return nil, err
 	}
 
-	cachedResp := c.cache.Get(cacheKey)
-	isCached := cachedResp != nil && !cachedResp.Expired()
-	span.SetAttributes(attribute.Bool("is_cached", isCached))
-	if isCached {
-		checkCacheHitCounter.Inc()
+	skipCache := c.enableConsistencyOptions && req.Consistency == openfgav1.ConsistencyPreference_HIGHER_CONSISTENCY
+
+	if !skipCache {
```

Comment:
```suggestion
	tryCache := !c.enableConsistencyOptions || req.Consistency == openfgav1.ConsistencyPreference_MINIMIZE_LATENCY

	if tryCache {
```

because double negation makes my brain hurt

Reply:
Heh I had been wanting to change this to avoid double negation but couldn't decide on the right wording. I think we might want to do `!= ConsistencyPreference_HIGHER_CONSISTENCY` as we want `ConsistencyPreference_UNSPECIFIED` to use the cache

Reply:
Done in c3c6d511e255f424d3c85e7a61f9e376cbc5f478, also added a test to cover the usage of `UNSPECIFIED` trying the cache